common/shlibs

```diff
@@ -3856,3 +3856,5 @@ libbudgie-private.so.0 budgie-desktop-10.5.1_1
 libbudgietheme.so.0 budgie-desktop-10.5.1_1
 libbudgie-plugin.so.0 budgie-desktop-10.5.1_1
 libslirp.so.0 libslirp-4.2.0_1
+libgala.so.0 gala-3.2.0_1
+libwingpanel-2.0.so.0 wingpanel-2.3.0_1
```

Comment:
Those should be part of the commit adding the package.

Reply:
Done!

Reply:
Done!